### ISSUE TITLE
line 51 is a wrong code? 

### ISSUE THREAD

**Issue by jiezhangGt** [kyegomez/LM-Infinite#5]
logits = logits = mask.to(logits.device)

or 

logits = logits + mask.to(logits.device)
